src/components/html/EcAdminSettingsForm.html

```diff
@@ -10,17 +10,16 @@
         <h5
           role="button"
           class="card-title bg-light"
-          @click="visibleCollapse = i"
+          v-b-toggle="'collapse-' + i"
         >
           {{ header }}
         </h5>
       </div>
-
       <b-collapse
         class="card-body"
         accordion="admin-settings"
         :id="'collapse-' + i"
-        :visible="visibleCollapse === i"
+        :visible="i == 0"
```

Comment:
Hmm, mas isso não vai forçar o primeiro **sempre visível** @thiagoabreudev ?

Reply:
Não

Reply:
@leomp12 , esse cara é somente para status inicial, veja: https://bootstrap-vue.org/docs/components/collapse#collapse

Reply:
Tendeu, API contraditória :stuck_out_tongue: